cipher/Cargo.toml

```
@@ -17,6 +18,7 @@ inout = "0.1"
 
 # optional dependencies
 blobby = { version = "0.3", optional = true }
+zeroize = { version = ">=1, <1.6", optional = true, default-features = false }
```

Realistically I don't foresee bumping `zeroize`'s MSRV above 1.56 any time in the foreseeable future (it's presently 1.51), so I think we can get rid of this pinning.

```suggestion
zeroize = { version = "1", optional = true, default-features = false }
```